src/map/pc.hpp

```diff
@@ -730,6 +730,15 @@ struct map_session_data {
 	uint32* hatEffectIDs;
 	uint8 hatEffectCount;
 #endif
+
+	struct {
+		unsigned attack   : 1;
```

Comment:
If we continue support for `unitblockmove` and `unitblockskill`, would attack be an option to throw into `unit.hpp::unit_data` too?

Reply:
If we implement a command to block the attack of a unit, eventually. For now no need to move it to `unit_data` I guess.

Reply:
`unitblockmove` and `unitblockskill` could be dropped. The options are supported largely by `setunitdata` if I'm not wrong.

Reply:
I know we can currently manipulate movement with `*_CANMOVE_TICK` and we have `*_IMMUNE_ATTACK` which isn't necessarily equal to blocked skill usage. But the `unit_data::state` could most definitely be added to `setunitdata`/`getunitdata`.